shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/DDLStatement.g4

```diff
@@ -1056,10 +1056,10 @@ foreignServerVersion
 
 alterStatistics
     : ALTER STATISTICS
-    ( ifExists? anyName SET STATISTICS signedIconst
```

Comment:
Why remove ifExists?

Reply:
it doesn't exsits in the PostgreSQL dialect here 
https://www.postgresql.org/docs/current/sql-alterstatistics.html

Reply:
> it doesn't exsits in the PostgreSQL dialect here https://www.postgresql.org/docs/current/sql-alterstatistics.html

Can you check this grammar in PostgreSQL source code?

Reply:
you mean the postgres **gram.y** file right?

```
AlterStatsStmt:
			ALTER STATISTICS any_name SET STATISTICS SignedIconst
				{
					AlterStatsStmt *n = makeNode(AlterStatsStmt);

					n->defnames = $3;
					n->missing_ok = false;
					n->stxstattarget = $6;
					$$ = (Node *) n;
				}
			| ALTER STATISTICS IF_P EXISTS any_name SET STATISTICS SignedIconst
				{
					AlterStatsStmt *n = makeNode(AlterStatsStmt);

					n->defnames = $5;
					n->missing_ok = true;
					n->stxstattarget = $8;
					$$ = (Node *) n;
				}
			;
```
There if exists is supported.

Reply:
Yes, document is sometimes inaccurate, and source code can give us better guidance.

Reply:
thank you for this tip! will keep this in my mind for my next PRs . Ill revert the changes for `DDLStatement.g4`